src/libtorch.cc

```diff
@@ -430,6 +430,9 @@ class ModelInstanceState : public BackendModelInstance {
   // that output in the model.
   std::unordered_map<std::string, int> output_index_map_;
   std::unordered_map<std::string, TRITONSERVER_DataType> output_dtype_map_;
+
+  // If the input to the tensor is a dictionary of tensors.
```

Comment:
This comment is confusing. Shouldn't it be "If the input to the model is a dictionary of tensors"?